bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ConfigExtractor.scala

```diff
@@ -264,7 +266,7 @@ object ConfigExtractor {
         fieldLens <- extract[FieldLens]     (c, AoFieldLensKey)
         wfsMode   <- extract[GuideStarType] (c, AoGuideStarTypeKey)
       } yield {
-        val separation = distance(targetEnv.getBase, guideStar, when)
+        val separation = distance(targetEnv.getAsterism, guideStar.getTarget, when)
```

Comment:
@andrewwstephens can you provide some insight?  This is sort of a theoretical question for the time being since GN / Altair isn't used with an instrument that has multiple targets.  If it were used with a multi-target instrument would it make sense to consider the distance between the center point of the asterism and the guide star for strehl calculations?  Perhaps AO and multiple targets wouldn't mix in this way in practice?

I found this calculation which seems relevant:

````java
    // Calculates the strehl noise from the guide star distance
    //  the return values is used to calculate the strehl
    public double getStrehlNoiseDist() {
        double dgs0, dgspow;

        if (fieldLensIsIn()) {
            dgs0 = 30d; //if FL in (FL always in for LGS)
            dgspow = 2d; //if FL in (FL always in for LGS)

        } else {
            dgs0 = 5d; //if FL out
            dgspow = 1d; //if FL out
        }
        //double strehlNoiseDistance = Math.exp(-1*Math.pow(guideStarDistance/12.5,2)*Math.pow(1650/wavelength,2));  //Old caclulation

        return Math.exp(-1 * Math.pow(altair.guideStarSeparation() / dgs0, dgspow) * Math.pow(1650 / wavelength, 2));
    }
````